### ISSUE TITLE
⛔ 🐞 Bug Report: Loop Grid Equal Height not working on CTA widget

### ISSUE THREAD

**Issue by puregraphx** [elementor/elementor#22818]
### Prerequisites

- [X] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [X] The issue still exists against the latest stable version of Elementor.

### Description

I have created a CPT via the Toolset plugin. I can query that CPT via the Loop Grid Widget.
I have used the CTA Widget inside the Loop Template and set the Loop Widget to Equal Height.
The height of the boxes is still different.
I am using Elementor 3.14.0 and Elementor Pro 3.14.0.

### Steps to reproduce

Prerequisites: add Customer Post Types with: post title, post content, Featured Image, CPT post intro) and make sure you have different Post Intro lengths.

1. Create a new page
2. Insert the Loop Grid 
3. Create a new template
4. Add the CTA Widget
5. Save the Template
6. View the page

![image](https://github.com/elementor/elementor/assets/10434048/b0a66ee3-a4a8-4f29-a217-d792a36e0477)
 ![image](https://github.com/elementor/elementor/assets/10434048/1a1ec3c2-871a-4b3d-aac5-bfea00517a34)

Only workaround I have found so far is to set the CTA Widget Box to a custom 100% height.
![image](https://github.com/elementor/elementor/assets/10434048/2ecc0069-029a-4bda-8a87-68ac2585e41d)

### Isolating the problem

- [ ] This bug happens with only Elementor plugin active (and Elementor Pro).
- [X] This bug happens with a Blank WordPress theme active ([Hello theme](https://wordpress.org/themes/hello-elementor/)).
- [X] I can reproduce this bug consistently following the steps above.

### System Info

<details>
<summary>System Info</summary>

````txt


					
== Server Environment ==
	Operating System: Linux
	Software: Apache/2.4.57 (Debian)
	MySQL version: mariadb.org binary distribution v10.4.20
	PHP Version: 8.0.28
	PHP Memory Limit: 512M
	PHP Max Input Vars: 10000
	PHP Max Post Size: 25M
	GD Installed: Yes
	ZIP Installed: Yes
	Write Permissions: All right
	Elementor Library: Connected

== WordPress Environment ==
	Version: 6.2.2
	Site URL: https://wordpress-348652-3593613.cloudwaysapps.com
	Home URL: https://wordpress-348652-3593613.cloudwaysapps.com
	WP Multisite: No
	Max Upload Size: 25 MB
	Memory limit: 40M
	Max Memory limit: 512M
	Permalink Structure: /%postname%/
	Language: en-US
	Timezone: Europe/Brussels
	Admin Email: [REDACTED]
	Debug Mode: Inactive

== Theme ==
	Name: Hello Elementor
	Version: 2.7.1
	Author: Elementor Team
	Child Theme: No

== User ==
	Role: administrator
	WP Profile lang: nl_NL
	User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/114.0.0.0 Safari/537.36

== Active Plugins ==
	Booking Calendar
		Version: 10.Business.Small.SingleSite.9.0
		Author: wpdevelop, oplugins

	Code Snippets
		Version: 3.4.0
		Author: Code Snippets Pro

	Elementor
		Version: 3.14.0
		Author: Elementor.com

	Elementor Pro
		Version: 3.14.0
		Author: Elementor.com

	Enable Media Replace
		Version: 4.1.2
		Author: ShortPixel

	JetTricks
		Version: 1.4.4
		Author: Crocoblock

	ShortPixel Image Optimizer
		Version: 5.2.3
		Author: ShortPixel - Convert WebP/AVIF & Optimize Images

	Simple Custom CSS and JS
		Version: 3.44
		Author: SilkyPress.com

	Toolset Types
		Version: 3.4.19
		Author: OnTheGoSystems


== Must-Use Plugins ==
	Object Cache Pro (MU)
		Version: 1.18.2
		Author: Rhubarb Group


== Features ==
	Custom Fonts: 0
	Custom Icons: 0

== Integrations ==
	


== Elementor Experiments ==
	Optimized DOM Output: Active
	Improved Asset Loading: Active by default
	Improved CSS Loading: Active by default
	Inline Font Icons: Active
	Accessibility Improvements: Active by default
	Additional Custom Breakpoints: Active by default
	admin_menu_rearrangement: Inactive by default
	Flexbox Container: Active
	Upgrade Swiper Library: Active by default
	Grid Container: Active
	Default to New Theme Builder: Inactive
	Hello Theme Header & Footer: Inactive
	Editor Top Bar: Active
	Landing Pages: Inactive
	Nested Elements: Inactive by default
	Nested Accordion: Inactive by default
	Lazy Load Background Images: Inactive by default
	Global Style Guide: Inactive by default
	Page Transitions: Inactive
	Notes: Inactive
	Loop: Active by default
	Form Submissions: Active by default
	Scroll Snap: Inactive
	Menu: Inactive by default


== Log ==
	
PHP: showing 6 of 6PHP: 2023-06-12 09:11:27 [warning X 2][/home/348652.cloudwaysapps.com/kmuyhtdvew/public_html/wp-content/plugins/elementor/app/modules/import-export/module.php::717] Attempt to read property "label" on null [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2023-06-12 09:11:28 [warning X 2][/home/348652.cloudwaysapps.com/kmuyhtdvew/public_html/wp-content/plugins/elementor/core/common/modules/ajax/module.php::175] Undefined array key "data" [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2023-06-12 13:17:32 [warning X 2][/home/348652.cloudwaysapps.com/kmuyhtdvew/public_html/wp-content/plugins/elementor/includes/managers/image.php::108] Trying to access array offset on value of type bool [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2023-06-12 13:17:36 [warning X 4][/home/348652.cloudwaysapps.com/kmuyhtdvew/public_html/wp-content/plugins/elementor/modules/history/revisions-manager.php::363] Trying to access array offset on value of type null [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2023-06-15 14:11:13 [error X 13][/home/348652.cloudwaysapps.com/kmuyhtdvew/public_html/wp-content/plugins/elementor/includes/managers/image.php::92] Uncaught Error: Undefined constant "Elementor\BFITHUMB_UPLOAD_DIR" in /home/348652.cloudwaysapps.com/kmuyhtdvew/public_html/wp-content/plugins/elementor/includes/managers/image.php:92
Stack trace:
#0 /home/348652.cloudwaysapps.com/kmuyhtdvew/public_html/wp-content/plugins/elementor/includes/managers/image.php(36): Elementor\Images_Manager->get_details()
#1 /home/348652.cloudwaysapps.com/kmuyhtdvew/public_html/wp-includes/class-wp-hook.php(308): Elementor\Images_Manager->get_images_details()
#2 /home/348652.cloudwaysapps.com/kmuyhtdvew/public_html/wp-includes/class-wp-hook.php(332): WP_Hook->apply_filters()
#3 /home/348652.cloudwaysapps.com/kmuyhtdvew/public_html/wp-includes/plugin.php(517): WP_Hook->do_action()
#4 /home/348652.cloudwaysapps.com/kmuyhtdvew/public_html/wp-admin/admin-ajax.php(188): do_action()
#5 {main}
  thrown [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2023-06-15 15:39:04 [warning X 2][/home/348652.cloudwaysapps.com/kmuyhtdvew/public_html/wp-content/plugins/elementor-pro/modules/theme-builder/classes/conditions-manager.php::171] Undefined array key "editor_post_id" [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]

Log: showing 15 of 152023-06-12 13:42:12 [info] elementor-pro::elementor_pro_updater Started 
2023-06-12 13:42:12 [info] Elementor Pro/Upgrades - _on_each_version Start  
2023-06-12 13:42:12 [info] Elementor Pro/Upgrades - _on_each_version Finished 
2023-06-12 13:42:12 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.11.6',
  'to' => '3.13.2',
)]
2023-06-12 13:42:12 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.11.6',
  'to' => '3.13.2',
)]
2023-06-19 12:04:22 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor',
  'from' => '3.13.4',
  'to' => '3.14.0',
)]
2023-06-19 12:04:22 [info] elementor-pro::elementor_pro_updater Started 
2023-06-19 12:04:22 [info] Elementor Pro/Upgrades - _on_each_version Start  
2023-06-19 12:04:22 [info] Elementor Pro/Upgrades - _on_each_version Finished 
2023-06-19 12:04:22 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.13.2',
  'to' => '3.14.0',
)]
2023-06-19 12:04:22 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.13.2',
  'to' => '3.14.0',
)]
2023-06-19 12:04:22 [info] elementor::elementor_updater Started 
2023-06-19 12:04:22 [info] Elementor/Upgrades - _on_each_version Start  
2023-06-19 12:04:22 [info] Elementor/Upgrades - _on_each_version Finished 
2023-06-19 12:04:22 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.13.4',
  'to' => '3.14.0',
)]



== Elementor - Compatibility Tag ==
	
	Elementor Pro: Compatible

== Elementor Pro - Compatibility Tag ==
	

				

````

</details>

**Comment by nicholaszein** [comment]
Hello @puregraphx,

Thank you for reaching out! 🙏 

### ⛔ While reviewing your report we concluded this is not an issue caused by Elementor. Here's what we found out:
Equal heights inside the Loop Grid or Loop Carousel apply to the **<ins>wrapper Container</ins>** of the `Loop Item` template, and not to the individual widgets inside it. Since there are N ways to build your templates, and N number of widgets you can add inside them, we can't simply apply equal heights to all widgets inside. This would create a cascading effect of an infinite Loop Item height. The limitation to apply it only to the `Container` helps you use it to your advantage to achieve the expected effect. We don't have any plans to change this behavior. See the video below to understand what I mean.

### ✅ What you can do to solve your issue:
- There are a few ways you can approach this. Here are two:
  1. Remove the background color from the CTA, and instead apply the background color to the wrapping **Container**; or simply apply the same background color of the CTA on the Container.
  2. Use the custom CSS snippet provided below on your CTA widget, so it can also adapt to the height of its wrapping Container.

```css

selector,
selector .elementor-cta {
    height: 100%;
}

```

See the video below to understand what I mean.


https://github.com/elementor/elementor/assets/113993460/00833e08-0922-4cdf-a40d-3704028edbfd



### 👍 For now, that's all we can help with. 

### ⚠️ Note that GitHub issues are reserved for Elementor 🐞 Bug Reports only, and should not be used for support.

❔ If you have questions, or need guidance, help, or general support, please use one of the following channels:

- [Elementor Global Community](https://facebook.com/groups/Elementors/) The main Facebook group where all kinds of users come together to help each other.
- [Elementor Academy](https://elementor.com/academy) Learn everything you need to know about using Elementor.
- [Elementor Developers Documentation](https://developers.elementor.com/) Learn everything about our Developer-API.
- [Free Support Forum](https://wordpress.org/support/plugin/elementor) Get help on WordPress.org.

💎 If you have an active [Elementor Pro](https://elementor.com/pro) license you are entitled to personal support, please see your purchase email or visit your [Elementor Account page](https://my.elementor.com/) for details.

Kind regards
